### DISCUSSION TITLE
Errors are incorrectly not displayed as stored in the cloud, some photos are stored in the cloud and are not visible in the app

### DISCUSSION THREAD

**Discussion by n4tur4lb0rnch1ll4** [immich-app/immich#9093] [General]
Hello everyone,

I have a Pixel 7 Pro and host Immich on my Synology NAS. Since one of the last updates, the last photos/videos are not shown as cloud-synced, but when I start it manually it says it was successfully uploaded - but they're still not there. However, I can see the photos in the web app on the computer. I also see photos there that are not synced to the phone.

Can someone please help me?

Many thanks and best regards

**Comment by alextran1502** [comment]
There are some sync issue at the moment and causing visual mismatch, logout and log back in should resolve it
